velox/dwio/parquet/reader/ParquetReader.cpp

```diff
@@ -286,6 +315,16 @@ std::unique_ptr<ParquetTypeWithId> ReaderBase::getParquetColumnInfo(
   if (isFileColumnNamesReadAsLowerCase()) {
     folly::toLowerAscii(name);
   }
+
```

Comment:
WHat if we just convert the name here, e.g. `name = updateName(tableSchema, schemaIdx, name)`? If that works then we don't need to push the name to columnNames array.